src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -212,12 +202,14 @@ trait ClusterComponent extends LeoComponent {
 
     def markPendingDeletion(googleId: UUID): DBIO[Int] = {
       clusterQuery.filter(_.googleId === googleId)
-        .map(c => (c.destroyedDate, c.status, c.hostIp))
-        .update(Timestamp.from(Instant.now()), ClusterStatus.Deleting.toString, None)
+        .map(c => (c.status, c.hostIp))
```

Comment:
Changed `completeDeletion` to set the `destroyedDate` instead of `markPendingDeletion`. A DELETING cluster should still be considered "active" because it's still being monitored.

`markPendingDeletion` still clears the `hostIp` so the notebook can no longer be proxied.